Template SSL check LLD.xml

```diff
@@ -196,7 +196,7 @@ SSL certificate on {#URL}:{#PORT} expires in less than 3 week</description>
                             <delay>1h</delay>
                             <history>10d</history>
                             <trends>0</trends>
-                            <value_type>CHAR</value_type>
+                            <value_type>1</value_type>
```

Comment:
In zabbix 4.4 - 5.0.3 export template write there CHAR, so I don't want change value there

Reply:
I didn't pay attention to the template I was importing at first and went for the one with a number. I did correct it after I realized that the other was the 4.4 template. :-)

Good point on the CHAR. I was just going for consistency. I will yank that part out of the pull request.

Thanks for your work on this template! It's very useful.

Reply:
Thanks for your contribution!